data/brands/shop/erotic.json

```diff
@@ -95,6 +95,18 @@
         "name:en": "Pink Rabbit",
         "shop": "erotic"
       }
+    },
+        {
+      "displayName": "Erotic Store Venus",
+      "locationSet": {"include": ["616"]},
+      "tags": {
+        "brand": "Erotic Store Venus",
+        "brand:wikidata": "Q122900757",
+        "name": "Erotic Store Venus",
+        "shop": "erotic"
+        "min_age": "18"
+        "website": "https://www.venus.net.pl/"
```

Comment:
Please note that we don't support websites in NSI (see [this wiki page](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required))
```suggestion
```

Reply:
Each tag also needs a comma at the end (except the last one) otherwise it causes a build error.